models/cow_protocol/ethereum/cow_protocol_ethereum_schema.yml

```diff
@@ -0,0 +1 @@
+# Kept here for future use
```

Comment:
I am still new to this, but I guess we don't need to define a schema for views? I will be using this file in a follow up PR for the incremental trades table.

Reply:
We always need a schema, even for views. This is quite important, since it's also used to generate spellbook documentation. Pls make sure to include column names, descriptions, tags, metadata, etc... here